cypress.json

```diff
@@ -1,3 +1,5 @@
 {
-  "baseUrl": "http://localhost:3000"
+  "screenshotsFolder": "tmp/cypress_screenshots",
```

Comment:
These are changes that got made by the cypress-rails gem. In a nutshell, it moves test artifacts to the `/tmp` folder and does not delete them between test runs. This is so we can view these on Travis if ever something goes wrong with an e2e test.